packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -104,6 +104,7 @@ export class NgccProcessor {
 }
 
 class NgccLogger implements Logger {
+  private alreadyProcessMsg = 'The target entry-point has already been processed';
```

Comment:
If `ngcc` change it's logging messages our filter will break. 

@petebacondarwin @gkalpak do you have a suggestion on a better way to log only when something is actually happening?

Reply:
How about we just agree to change the `info` messages that you do not wish to see to `debug` messages? Then you don't need to filter at all?

Reply:
But I think it would also be fair for CLI to output a message before ngcc is triggered to say something like "Compiling Angular packages with ngcc".

Reply:
It's hard to go down that route because the CLI doesn't know if ngcc will try to compile a package or skip it. If we had such message it would come up on every build, several times.